molecule/default/molecule.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+dependency:
+  name: galaxy
+driver:
+  name: default
+  options:
+    managed: false
+    ansible_connection_options:
+      ansible_connection: local
+platforms:
+  - name: instance
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      callbacks_enabled: ansible.posix.profile_tasks
+  inventory:
+    hosts:
+      all:
+        hosts:
+          instance:
+            ansible_host: localhost
+    host_vars:
+      instance:
+        target: "localhost"
+        kubeconfig_path: "$HOME/.kube/config"
```

Comment:
I don't think this will technically work, or will it, I wonder if it's being resolved by Molecule to /root/ instead of by Ansible.

I wonder if we can set KUBECONFIG as an environment variable and let `kubeconfig_path` point to that environment variable?